activation/poet.go

```diff
@@ -308,7 +308,7 @@ func (c *HTTPPoetClient) req(ctx context.Context, method, path string, reqBody,
 	}
 
 	if res.StatusCode != http.StatusOK {
-		c.logger.Info("got poet response != 200 OK", zap.String("status", res.Status), zap.String("body", string(data)))
+		c.logger.Warn("got poet response != 200 OK", zap.String("status", res.Status), zap.String("body", string(data)))
```

Comment:
Arguably this should be debug - This isn't really information relevant to the user

Reply:
I turned it into a debug because it might be important for the user to know why his requests to the poet are rejected (e.g. rate-limited, rejected by firewall etc.).

Reply:
How about making this log line `Debug` and printing more use friendly logs in the `switch` further below, i.e. "PoET request failed" for 400 and "Failed PoET authorization" for 401.

Reply:
After giving it more thought, changed it to DEBUG. The caller has all the necessary information in the returned error.